crypto/x509/v3_ncons.c

```diff
@@ -34,9 +34,12 @@ static int nc_match_single(GENERAL_NAME *sub, GENERAL_NAME *gen);
 static int nc_dn(X509_NAME *sub, X509_NAME *nm);
 static int nc_dns(ASN1_IA5STRING *sub, ASN1_IA5STRING *dns);
 static int nc_email(ASN1_IA5STRING *sub, ASN1_IA5STRING *eml);
+static int nc_email_eai(ASN1_UTF8STRING *sub, ASN1_IA5STRING *eml);
 static int nc_uri(ASN1_IA5STRING *uri, ASN1_IA5STRING *base);
 static int nc_ip(ASN1_OCTET_STRING *ip, ASN1_OCTET_STRING *base);
 
+int a2ulabel(const char *in, char *out, size_t *outlen); /* FIXME beldmit */
```

Comment:
What is the FIXME?

Reply:
The FIXME here means we need to move the declaration of this function to a header file. Will do.

Reply:
Fixed.